src/Microsoft.SqlTools.ServiceLayer/TableDesigner/TableDesignerValidator.cs

```diff
@@ -518,8 +519,9 @@ public List<TableDesignerIssue> Run(Dac.TableDesigner designer)
             {
                 errors.Add(new TableDesignerIssue()
                 {
-                    Description = "The table has more than one edge constraint on it. This is only useful as a temporary state when modifying existing edge constraints, and should not be used in other cases. Please refer to https://docs.microsoft.com/sql/relational-databases/tables/graph-edge-constraints for more details.",
```

Comment:
I also updated this one using `MoreInfoLink` as I was making related chagnes. I've tested this one and the link is working correctly,